srcpkgs/oneko/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'oneko'
+pkgname=oneko
+version=1.2.5
+revision=1
+# other archs end with /usr/bin/xmkmf: line 53: 15231 Aborted
+archs="x86_64"
+_pkgver=1.2.sakura.5
+wrksrc=${pkgname}-${_pkgver}
+build_style=gnu-configure
+hostmakedepends="imake gccmakedep"
+makedepends="libX11-devel libXext-devel"
+short_desc="Cat that chases around your cursor"
+maintainer="wael <40663@protonmail.com>"
+license="Public Domain"
+homepage="http://www.daidouji.com/oneko"
+distfiles="${homepage}/distfiles/${pkgname}-${_pkgver}.tar.gz"
+checksum=2c2e05f1241e9b76f54475b5577cd4fb6670de058218d04a741a04ebd4a2b22f
+
+do_configure() {
+	xmkmf -a
+}
+
+post_install() {
+	vman oneko.man oneko.1
+	vmkdir usr/share/man/jp/man1/oneko.1
+	mv ${wrksrc}/oneko.man.jp ${DESTDIR}/usr/share/man/jp/man1/oneko.1
```

Comment:
```suggestion
	vman oneko.man.jp oneko.jp.1
```
should work for that

Reply:
i assumed that `vman` automatically just took the 2nd argument and added it in without anything special.
i assume whats happening here is that by adding `jp` to the file name vman will automatically move to to `jp/man1`